front/app/containers/IdeasNewPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -81,10 +81,13 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
   const [ideaId, setIdeaId] = useState<string | undefined>();
 
   const [initialFormData, setInitialFormData] = useState({});
-  const participationContext = getCurrentPhase(phases?.data);
```

Comment:
Noticed that this was not needed as we need the anonymousPosting attribute from the phase the form is displaying for - which for an admin might not be the current phase